index.html

```diff
@@ -2249,6 +2246,33 @@ <h3>
           available underneath an icon).
         </p>
       </section>
+      <section>
+        <h3 id="installation-sec">
+          Launching a web application
+        </h3>
+        <p>
+          The steps to <dfn>launch a web application</dfn> is given by the
+          following algorithm. The algorithm takes a
+          [=Document/processed manifest=] |manifest:processed manifest|,
+          an optional |target URL:URL| and returns a
+          [=service worker client/window client=].
+        </p>
+        <ol class="algorithm">
+          <li>If <var>url</var> is unset, set <var>url</var> to
+              |manifest|.[=manifest/start_url=].
+          </li>
+          <li>Let |client| be the result of
+              <a href="https://html.spec.whatwg.org/multipage/browsers.html#creating-a-new-top-level-browsing-context">
+              creating a new top-level browsing context</a>.
+          </li>
+          <li>[=Apply=] |manifest| to |client|.
+          </li>
+          <li>[=Navigate=] |client| to |url|.
+          </li>
+          <li>Return |client|.
```

Comment:
Navigating takes time... does that matter? Also, need to check if navigation can fail. 

Also, does the client need to be returned?

Reply:
For this particular case default navigation failure handling is sufficient.

Removed the client return.